### PR TITLE
Review edit

### PR DESCRIPTION
**Pull request by Aferencz1987**
## Changes
Added review edit with its testing
## Checklist
- [x] Tests written where required
- [x] All tests passing
- [x] SimpleCov at 95%

closes #

### CODE REVIEW

File: app/graphql/mutations/review_edit.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      body ||= review.body
+      title ||= review.title
+      gender ||= review.gender
+      size ||= review.size
+      payment ||= review.payment
+      extended_body ||= review.extended_body
+      kindness ||= review.kindness
+      vibe ||= review.vibe
+      date_again ||= review.date_again
+      gender ||= review.gender
```
Comment (LawrenceWhalen) [inline app/graphql/mutations/review_edit.rb:44]: Look into a cleaner way to do this later, but I am glad the single liners work.

File: spec/graphql/mutation/review_edit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```
Comment (LawrenceWhalen) [inline spec/graphql/mutation/review_edit_spec.rb:1]: Yay for TDD

**Review by LawrenceWhalen — APPROVED** [review]
As always, a pleasure to driver/navigator with you.

File: spec/graphql/mutation/review_edit_spec.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                size
+                body
+                safetyMeter
+                rating
+              }
+            }
+          }
+        GQL
+    end
+
```
Comment (ahenzler) [inline spec/graphql/mutation/review_edit_spec.rb:50]: remove whitespace

**Review by ahenzler — APPROVED** [review]

**Pull request merged**